### ISSUE TITLE
Moduleise the paddle funcs

### ISSUE THREAD

**Issue by MarkEdmondson1234** [MarkEdmondson1234/Shiny-R-SaaS#3]
Or functions

**Comment by GitHunter0** [comment]
@MarkEdmondson1234 , hey man, your project is exactly what I was looking for! 

I expected to finish an initial version of my Shiny App next month and I was hoping to find a way to monetize it because that income would help me a lot. I hope to be able to make my app work in your template. 

Thank you very much for the effort! It is very appreciated.

PS: Sorry if my message is misplaced here, I didn't know any other way to contact you. Cheers my friend

**Comment by MarkEdmondson1234** [comment]
No worries, did you come here via the blog post? https://code.markedmondson.me/datascience-aas/ 

A comment on there is more usual, but no worries.

**Comment by GitHunter0** [comment]
@MarkEdmondson1234 , yes, now I saw the comments section, I was in a hurry and did not see it. I will delete the comment here and later reach to you there. Thanks again
